code/stable/glassbr/SRS/GlassBR_SRS.tex

```diff
@@ -481,7 +481,7 @@ \subsubsection{Data Definitions}
         
 \\ \midrule \\
 Equation & \begin{displaymath}
-           B=\frac{k}{\left(a b\right)^{m-1}} \left(E h^{2}\right)^{m} LDF e^{J}
+           B=\frac{k}{\left(a b\right)^{m-1.0}} \left(E h^{2}\right)^{m} LDF e^{J}
```

Comment:
this is likely another case where the exponent might be exact - not sure.

Reply:
I _think_ you're correct in that it should be an exact as well. I changed it for now, we can alter it later if we would like.

Reply:
Yes, I agree that is should be exact.  I believe that the (m-1) is assumed and then fitting of experimental data is done to find the values for m.  I could look into this in more detail, but the GlassBR example isn't one that will have its knowledge spread to many future examples.  I don't think it is something we need to worry about.